modules/customize-certificates-add-service-serving-configmap.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-configmap-{context}"]
+= Add a service certificate to a ConfigMap
+
+To secure communication to services that use a ConfigMap, 
+have the cluster generate a signed serving certificate and key pair into the
+ConfigMap. To do this, set the 
+`service.beta.openshift.io/inject-cabundle=true` 
+annotation on your ConfigMap. A certificate will be generated, which allows
```

Comment:
The CA is not generated - the configmap is injected with the current service CA certificate (it is generated elsewhere automatically, though this is not something to document).

Reply:
This has been corrected.